### ISSUE TITLE
[Table Accessibility Issue]: Appendix 1: UKHO Checklist

### ISSUE THREAD

**Issue by jooyoungseo** [gvwilson/10-accessible#7]
Two screen-reader accessibility issues found in Table "Appendix 1: UKHO Checklist":

1. `colspan` attribute should be avoided if possible (don't merge cells if possible): this may confuse screen reader users' cognitive and keyboard navigation. Tidy data would be preferrable.

2. Don't bind data frames if possible: We may want to split the combined one table into each disability category. Because `<th>` tag was used mutiple times within the same table, screen readers (e.g., JAWS and NVDA) recognize them as multi-level headers, which is misleading.

I will make a PR to close this issue.

BTW, we may want to include table a11y in our tips. Just a thought.
